[ADD] Recruitment: Team performance reporting (#10984)
odoo/documentation: content/applications/hr/recruitment/team_performance.rst

==========================
Team performance reporting
==========================

The *Team Performance* report in the **Recruitment** app shows how many applicants each recruiter is
managing.

The information is determined by the person that populates the :ref:`recruiter field
<recruitment/applicant-details>` on each applicant form.

Open report
===========

To access the *Team Performance* report, navigate to :menuselection:`Recruitment app --> Reporting
--> Team Performance`.

This number of :guilabel:`In Progress`, :guilabel:`Hired`, and :guilabel:`Refused` applicants for
each recruiter is displayed in a default :icon:`fa-area-chart` :guilabel:`(Graph)` view. The

> information shown is for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`.

Reviewer: ```suggestion
information shown is for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`, as displayed in the search bar.
```

Pivot table view
----------------

For a more detailed view of the information in the :guilabel:`Team Performance` report, click the
:icon:`oi-view-pivot` :guilabel:`(Pivot)` icon in the top-right corner. This displays all the
information in a pivot table.

In this view, the job positions are displayed in the rows, while the columns display the total
number of applicants. The applicants are further organized by :guilabel:`# Applicant` (in process),
:guilabel:`# Hired`, and :guilabel:`# Refused`.

The displayed information can be modified, if desired.

In this example, there are 19 total applicants. Out of those 19, eight have been hired, and three
refused.

From the data presented, the :guilabel:`Experienced Developer` job position is the most successful.
This job position has the highest number of total applicants, as well as the most hires. In
addition, the :guilabel:`Experienced Developer` has the least amount of refused applicants.

This pivot table also shows that the :guilabel:`Chief Executive Officer` position is the hardest to
fill, as it has the fewest total applicants.

.. image:: team_performance/team-perf-pivot.png
   :align: center

>    :alt: The detailed pivot table view.

Reviewer: This might look good in an example admonition, but I can also see leaving it out so it doesn't segment it from the following use case. Just something to consider.

Author: In the reporting docs I've done, I have made a use case in each of them to show what the report can do, and I think it was suggested to have it as a section instead of an admonition, so that's why I took it out of an example block. But I appreciate the suggestion!

   :alt: The detailed pivot table view.

> Use case: recruiter performance over time
> ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Reviewer 2: Move this heading up from an h4 to an h2. The use case is an important part of the reporting doc, and I think it should be highlighted in the in page TOC :)
```suggestion
Use case: recruiter performance over time
=========================================
```

One way to modify this report is to show how recruiters are performing over time. To show this
information, begin with the :guilabel:`Team Performance` report in the :icon:`oi-view-pivot`
:guilabel:`(Pivot)` view.

> Next, click the :icon:`oi-caret-down` :guilabel:`(down caret)` in the search bar, revealing a

Reviewer: ```suggestion
Next, click the :icon:`fa-caret-down` :guilabel:`(down caret)` in the search bar, revealing a
```

Next, click the :icon:`fa-caret-down` :guilabel:`(down caret)` in the search bar, revealing a
drop-down menu. Click :guilabel:`Add Custom Group` :icon:`oi-caret-down` at the bottom of the
:icon:`oi-group` :guilabel:`Group By` column, then click :guilabel:`Recruiter`. Click away from the
drop-down menu to close it. Now, each row on the table represents a recruiter.

.. image:: team_performance/by-recruiter.png
   :align: center
   :alt: The pivot table now displaying the recruiters in the rows.

To compare the team's performance over different time periods, click the :icon:`fa-caret-down`

> :guilabel:`(down arrow)` in the search bar. Click :guilabel:`Start Date` :icon:`oi-caret-down` in

Reviewer 2: Fix icon class
```suggestion
:guilabel:`(down arrow)` in the search bar. Click :guilabel:`Start Date` :icon:`fa-caret-down` in
```

the :icon:`fa-filter` :guilabel:`Filters` column, revealing various time periods to select.

> In this example, the desired data is to compare the team's performance in the third quarter (June -

Reviewer: ```suggestion
In this example, the desired data is the comparison between the team's performance in the third quarter (June -
```

==========================
Team performance reporting
==========================

The *Team Performance* report in the **Recruitment** app shows how many applicants each recruiter is
managing.

This information is determined by the persons populating the :ref:`Recruiter
<recruitment/applicant-details>` field on each applicant form.

Open report
===========

To access the *Team Performance* report, navigate to :menuselection:`Recruitment app --> Reporting
--> Team Performance`.

The number of :guilabel:`In Progress`, :guilabel:`Hired`, and :guilabel:`Refused` applicants for

> each recruiter is displayed in the :icon:`fa-area-chart` :guilabel:`(Graph)` view, by default. The
> information shown is for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant`, as displayed in
> the search bar.

Reviewer 3: ```suggestion
each recruiter is displayed in the default :icon:`fa-area-chart` :guilabel:`(Graph)` view.

The information shown is for the :icon:`fa-filter` :guilabel:`Last 365 Days Applicant` default filter, as displayed in
the search bar.
```

> Hover the cursor over any column to view a popover window displaying the specific details for that

Reviewer 3: ```suggestion
Hover the cursor over any column to view a popover window, displaying the specific details for that
```
OPTIONAL: added comma for enhanced sentence pacing

number of applicants. The applicants are further organized by :guilabel:`# Applicant` (in process),
:guilabel:`# Hired`, and :guilabel:`# Refused`.

The displayed information can be modified, if desired.

In this example, there are 19 total applicants. Out of those 19, eight have been hired, and three
refused.

From the data presented, the :guilabel:`Experienced Developer` job position is the most successful.
This job position has the highest number of total applicants, as well as the most hires. In
addition, the :guilabel:`Experienced Developer` has the least amount of refused applicants.

This pivot table also shows that the :guilabel:`Chief Executive Officer` position is the hardest to
fill, as it has the fewest total applicants.

.. image:: team_performance/team-perf-pivot.png
   :align: center
   :alt: The detailed pivot table view.

Use case: recruiter performance over time
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

One way to modify this report is to show how recruiters are performing over time. To show this
information, begin with the :guilabel:`Team Performance` report in the :icon:`oi-view-pivot`
:guilabel:`(Pivot)` view.

> Next, click the :icon:`fa-caret-down` :guilabel:`(down caret)` in the search bar, revealing a

Reviewer 3: ```suggestion
Next, click the :icon:`fa-caret-down` :guilabel:`(down arrow)` in the search bar, revealing a
```
SUGGESTION: I believe the term "arrow" is more universally understood compared to 'caret.' I know that's the technical term, but it might be easier to use "arrow" for an increased chance of international comprehension.

drop-down menu. Click :guilabel:`Add Custom Group` :icon:`oi-caret-down` at the bottom of the

> :icon:`oi-group` :guilabel:`Group By` column, then click :guilabel:`Recruiter`. Click anywhere on
> the screen to close the drop-down menu. Now the rows change from the job positions to the
> recruiters.

Reviewer 3: ```suggestion
:icon:`oi-group` :guilabel:`Group By` column, then click :guilabel:`Recruiter`. Click away from the drop-down menu to
close it. Now, each row on the table represents a recruiter.
```

.. image:: team_performance/by-recruiter.png
   :align: center
   :alt: The pivot table now displaying the recruiters in the rows.

To compare the team's performance over different time periods, click the :icon:`fa-caret-down`

> :guilabel:`(down caret)` in the search bar. Click :guilabel:`Start Date` :icon:`oi-caret-down` in

Reviewer 3: ```suggestion
:guilabel:`(down arrow)` in the search bar. Click :guilabel:`Start Date` :icon:`oi-caret-down` in
```
same reasoning as above

the :icon:`fa-filter` :guilabel:`Filters` column, revealing various time periods to select.

In this example, the desired data is the comparison between the team's performance in the third

> quarter (June - August) with the second quarter (April - July). To do so, click :guilabel:`Q3`. Once

Reviewer 3: ```suggestion
quarter (June - August) and the second quarter (April - July). To do so, click :guilabel:`Q3`. Once
```

> clicked, the current year is also ticked, in this example, :guilabel:`2024`.

Reviewer 3: ```suggestion
clicked, the current year is also ticked. In this example, it is :guilabel:`2024`.
```

After making this selection, a :icon:`fa-adjust` :guilabel:`Comparison` column appears. Click
:guilabel:`Start Date: Previous Period` to compare the third quarter with the second quarter, for
the various recruiters.

.. image:: team_performance/compare.png
   :align: center
   :alt: A comparison table of recruiter totals of Q2 and Q3.

From this report, several things can be extrapolated: the total number of applicants increased, the
number of hired applicants remained the same, while the number of refused applicants decreased.

> Additionally, :guilabel:`Jane Jobs` had the highest increase in numbers of applicants in the third

Reviewer 3: ```suggestion
Additionally, :guilabel:`Jane Jobs` had the highest increase in number of applicants during the third
```

quarter, but her number of hired applicants went down :guilabel:`67%`. The recruiter with the best

> overall numbers was :guilabel:`Rose Recruiter`, who had both their active applicants and hired
> applicants increase in the third quarter, while their refused applicants went down.

Reviewer 3: ```suggestion
overall numbers was :guilabel:`Rose Recruiter`, who had both, their active applicants and hired
applicants, increase in the third quarter, while their refused applicants went down.
```
